docs/guide/data_types/timeseries.rst

```diff
@@ -296,123 +189,95 @@ You can even change all the values for a given time: ::
+It's common to want to combine a number of TimeSeries together into a single TimeSeries.
+In the simplest scenario this is to combine data from a single source over several time ranges, for example if you wanted to combine the daily GOES data to get a week or more of constant data in one TimeSeries.
+This can be performed using the TimeSeries factory with the ``concatenate=True`` keyword argument: ::
 
     >>> concatenated_timeseries = sunpy.timeseries.TimeSeries(filepath1, filepath2, source='XRS', concatenate=True)  # doctest: +SKIP
 
-Note, you can list any number of files, or a folder or use a glob to select the
-input files to be concatenated.  It is possible to concatenate two TimeSeries
-after creating them with the factory using the `~sunpy.timeseries.GenericTimeSeries.concatenate`
-method.  For example: ::
+Note, you can list any number of files, or a folder or use a glob to select the input files to be concatenated.
+It is possible to concatenate two TimeSeries after creating them with the factory using the `~sunpy.timeseries.GenericTimeSeries.concatenate` method.
+For example: ::
 
     >>> concatenated_timeseries = goes_timeseries_1.concatenate(goes_timeseries_2)  # doctest: +SKIP
 
-This will result in a TimeSeries identical to if you used the factory to create
-it in one step.  A limitation of the TimeSeries class is that often it is not
-easy to determine the source observatory/instrument of a file, generally
-because the file formats used vary depending on the scientific working groups,
-thus some sources need to be explicitly stated (as a keyword argument) and so it
-is not possible to concatenate files from multiple sources with the factory.
-To do this you can still use the `~sunpy.timeseries.GenericTimeSeries.concatenate`
-method, which will create a new TimeSeries with all the rows and columns of the
-source and concatenated TimeSeries in one: ::
+This will result in a TimeSeries identical to if you used the factory to create it in one step.
+A limitation of the TimeSeries class is that often it is not easy to determine the source observatory/instrument of a file, generally because the file formats used vary depending on the scientific working groups, thus some sources need to be explicitly stated (as a keyword argument) and so it is not possible to concatenate files from multiple sources with the factory.
+To do this you can still use the `~sunpy.timeseries.GenericTimeSeries.concatenate` method, which will create a new TimeSeries with all the rows and columns of the source and concatenated TimeSeries in one: ::
 
     >>> concatenated_timeseries = goes_timeseries.concatenate(eve_timeseries)  # doctest: +SKIP
 
-Note that the more complex `~sunpy.timeseries.TimeSeriesMetaData`
-object now has 2 entries and shows details on both: ::
+Note that the more complex `~sunpy.timeseries.TimeSeriesMetaData` object now has 2 entries and shows details on both: ::
 
     >>> concatenated_timeseries.meta  # doctest: +SKIP
```

Comment:
A general point maybe to consider for a future PR is that I don't like how many of these are skipped. I would prefer working examples one day for the timeseries docs.